docs/tools/lsp-factoryjs/classes/03-lsp7-digital-asset.md

```diff
@@ -0,0 +1,181 @@
+---
+sidebar_position: 3
+title: LSP7DigitalAsset
+---
+
+# LSP7DigitalAsset
+
+## deploy
+
+```js
+lspFactory.LSP7DigitalAsset.deploy(
+  digitalAssetDeploymentOptions,
+  contractDeploymentOptions?
+);
+```
+
+The function deploys a mintable [LSP7 Digital Asset](../../../standards/nft-2.0/LSP7-Digital-Asset).
+
+#### Parameters
+
+| Name                            | Type   | Description                                          |
+| :------------------------------ | :----- | :--------------------------------------------------- |
+| `digitalAssetDeploymentOptions` | Object | The [constructor parameters] used when deploying.    |
+| `contractDeploymentOptions?`    | Object | The Specification for [Contract Deployment Options]. |
+
+#### Parameters of `digitalAssetDeploymentOptions`
+
+| Name                    | Type                               | Description                                                           |
+| :---------------------- | :--------------------------------- | :-------------------------------------------------------------------- |
+| `name`                  | string                             | The name of the token.                                                |
+| `symbol`                | string                             | The symbol of the token.                                              |
+| `controllerAddress`     | string                             | The owner of the contract.                                            |
+| `isNFT`                 | boolean                            | Specify if the contract represent a fungible or a non-fungible token. |
```

Comment:
```suggestion
| `isNFT`                 | boolean                            | Specify whether the token should be fungible by setting decimals value to 18. |
```

Reply:
A fungible token does not need to have 18 decemals. anyone can decide how much decimals he would like to have

Reply:
The `isNFT` constructor in LSP7 sets the decimals value to 18. See https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP7DigitalAsset/LSP7DigitalAssetCore.sol#L69, https://github.com/lukso-network/lsp-smart-contracts/blob/9469a23d2fa5fb71f2191895783dc34179e1306e/docs/ILSP7DigitalAsset.md#decimals

Reply:
You can maybe link to this example from the standard doc

Reply:
nice! did not know it was set in the standard